pkg/proxyagent/agent/manifests/charts/addon-agent/templates/addon-agent-role.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: rbac.authorization.k8s.io/v1
```

Comment:
Adding role and rolebinding in manifest work may dup with https://github.com/open-cluster-management-io/cluster-proxy/blob/b8a5476fb3162cdec8bf85c4515fdd40b645a94f/pkg/proxyagent/agent/agent.go#L75

Reply:
I think it's not a dup, since [cluster-proxy/pkg/proxyagent/agent/agent.go](https://github.com/open-cluster-management-io/cluster-proxy/blob/b8a5476fb3162cdec8bf85c4515fdd40b645a94f/pkg/proxyagent/agent/agent.go#L75-L102) is setting permission for the **HUB** cluster instead of the managed cluster.

Reply:
I got it. You're right about this.